### PR TITLE
feat: forgot and reset password[TAS-30][TAS-31]

### PR DESCRIPTION
**Pull request by femacedogomes**

### CODE REVIEW

File: src/app/forgotPassword/FormComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  });
+
+  const initialValues: IForgotPassword = {
+    email: "",
+  };
+
+  const handleFormSubmit = async (values: IForgotPassword) => {
+    setLoading(true);
+    try {
+        await Auth.forgotPassword({
```
Comment (pedrohbl03) [inline src/app/forgotPassword/FormComponent.tsx:23]: Show comments on [this PR](https://github.com/Agility-Creative-Solutions/ez2live-app/pull/14)

**Review by pedrohbl03 — CHANGES_REQUESTED** [review]
Resolve all requests appointed in PR from login page here to.

File: src/app/resetPassword/[token]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      .required("Confirme sua senha.")
+      .oneOf([Yup.ref("password")], "Senhas devem ser iguais."),
+  });
+
+  const initialValues: IResetPassword = {
+    password: "",
+    conf_password: ""
+  };
+
+  const handleFormSubmit = async (values : any) => {
```
Comment (pedrohbl03) [inline src/app/resetPassword/[token]/page.tsx:36]:
```suggestion
  const handleFormSubmit = async (values : Partial<IResetPassword>) => {
```

File: src/app/forgotPassword/FormComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  });
+
+  const initialValues: IForgotPassword = {
+    email: "",
+  };
+
+  const handleFormSubmit = async (values: IForgotPassword) => {
+    setLoading(true);
+    await Auth.forgotPassword({email: values.email,})
+    .then(()=> {
```
Comment (pedrohbl03) [inline src/app/forgotPassword/FormComponent.tsx:23]:
```suggestion
    .then((response)=> {
```

File: src/app/forgot-password/page.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import FormComponent from "./FormComponent";
+
+const UserRegisterPage = () => {
+
+  return (
+    <div className={`nc-PageSignUp `} data-nc-id="PageSignUp">
+      <div className="container mb-8 lg:mb-32">
+      <div className='mt-8 mb-16 flex items-center justify-between'>
+        <h2 className=" pl-6 flex items-center text-2xl leading-[115%] md:text-5xl md:leading-[115%] font-bold text-black dark:text-neutral-100 justify-center">
+          Recuperar <br/>
```
Comment (pedrohbl03) [inline src/app/forgot-password/page.tsx:13]: What is broke line here?
```suggestion
          Recuperar
```

**Review by pedrohbl03 — CHANGES_REQUESTED** [review]
@femacedogomes Resolve the comment where u set a "Broken line" pls.

File: src/app/forgot-password/page.tsx (hunk 4 above)
Comment (femacedogomes) [reply src/app/forgot-password/page.tsx:13]: its only for design

**Review by pedrohbl03 — APPROVED** [review]
gj

**Pull request merged**